integration/indexes_compat_test.go

```diff
@@ -342,13 +350,6 @@ func TestDropIndexesCompat(tt *testing.T) {
 			},
 			dropIndexName: "v_-1",
 		},
-		"AsteriskWithDropOne": {
```

Comment:
This one is checked by the driver itself. I remove the test case as it rather tests how driver works, not how FerretDB works.